src/StatusQ/Components/StatusChatList.qml

```diff
@@ -96,31 +94,27 @@ Column {
                 StatusChatListItem {
 
                     id: statusChatListItem
-
                     property string profileImage: ""
-
                     opacity: dragSensor.active ? 0.0 : 1.0
                     Component.onCompleted: {
                         if (typeof statusChatList.profileImageFn === "function") {
                             profileImage = statusChatList.profileImageFn(model.chatId || model.id) || ""
                         }
                     }
                     originalOrder: model.position
-                    chatId: model.chatId || model.id
-                    categoryId: model.categoryId || ""
+                    chatId: model.itemId || model.id
+                    categoryId: model.itemId || ""
                     name: !!statusChatList.chatNameFn ? statusChatList.chatNameFn(model) : model.name
-                    type: model.chatType
+                    type: !!model.type ? model.type : StatusChatListItem.Type.CommunityChat
                     muted: !!model.muted
-                    hasUnreadMessages: !!model.hasUnreadMessages || model.unviewedMessagesCount > 0
-                    hasMention: model.mentionsCount > 0
-                    badge.value: model.chatType === StatusChatListItem.Type.OneToOneChat ?
-                        model.unviewedMessagesCount || 0 :
-                        model.mentionsCount || 0
-                    selected: (model.chatId || model.id) === statusChatList.selectedChatId
+                    hasUnreadMessages: !!model.hasNotification || model.unviewedMessagesCount > 0
+                    hasMention: model.notificationsCount > 0
+                    badge.value: model.notificationsCount
+                    selected: (model.itemId || model.id) === statusChatList.selectedChatId
 
                     icon.color: model.color || ""
                     image.isIdenticon: !!!profileImage && !!!model.identityImage && !!model.identicon
-                    image.source: profileImage || model.identityImage || model.identicon || ""
+                    image.source: profileImage || model.icon || model.identicon || ""
```

Comment:
ok. how do we know if it's identicon?

Reply:
@alexandraB99 qml actually doesn't need to know what is it, whether it's an image or identicon, whether is a profile image or identity image or whatever, qml just need to display what backend says to display (with minimal component related logic), so you just need to set `icon` as a source and don't worry, if displayed image is not the correct one then we know it's up to backend and will fix it there.

See [here](https://github.com/status-im/StatusQ/blob/93563fc22854d7375bf2290e2f73c96707c33921/sandbox/Models.qml#L10) what we have for a model, `name`, `icon`, `color`, the same we have in our real backend model, so your component logic should be:

- if `model.icon` is not an empty string display icon
- if `model.icon` is an empty string display icon letter icon, using the first letter from `model.name` and color defined by `model.color`

Reply:
We do need to know though, if it's an identicon or not because the visuals change (there's a border rendered around the image, which is not the case when it's a normal profile picture). So we need some flag to know.

Various ways to go about this:

- Either introduce an `isIdenticon` flag,
- or introduce a more complex object like `model.image` which has all the information attached to it , similar to how it's set in QML

Reply:
@PascalPrecht thanks for this. 

> Either introduce an `isIdenticon` flag

This sounds ok, and easier do to than exposing an object in Nim. Will add it soon.